portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -9,7 +9,7 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
-import com.google.sps.data.Information;
+import com.google.sps.data.PostInformation;
```

Comment:
Wait, I don't see a class named PostInformation, only GetInformation. Am I missing something? Is this a class that came with the starter code that I didn't notice?